fbchat/graphql.py

```diff
@@ -26,12 +26,11 @@ def decode(self, s, _w=WHITESPACE.match):
 def graphql_color_to_enum(color):
     if color is None:
         return None
-    if len(color) == 0:
+    if not color:
         return ThreadColor.MESSENGER_BLUE
```

Comment:
I think this line won't execute because there is a None check before this one (or is the `not color` intended to catch another value?)

Reply:
The `not color` is intended to catch the empty string, but I guess that could be clearer